### ISSUE TITLE
Timeout not triggering on python find_any()

### ISSUE THREAD

**Issue by igor-markovic** [odriverobotics/ODrive#636]
**Describe the bug**

Python find_any() function doesn't return or raise exception after timeout and keeps blocking

**To Reproduce**
1. Open terminal
2. `$ python3`
3. `import odrive`
4. `driver = odrive.find_any(serial_number="204141512515", timeout=30)`
 Note: intentionally using some random serial that it shouldn't be able to find
5. Wait `timeout` seconds
6. function is still blocking

**Expected behavior**
find_any() function stops blocking when timeout is hit, raising `TimeoutError`

**Desktop (please complete the following information):**
 - OS: Linux (Ubuntu 20.04)
 - odrivetool Version: `v0.5.2.dev`

**Additional context**
When I hit ^C while the function is running it shows:
```
^CTraceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/Documents/ODrive/tools/odrive/__init__.py", line 101, in find_any
    current_timeout = None if timeout is None else min(0, timeout - (time.monotonic() - wait_start))
KeyboardInterrupt
```

Tested on another computer and it seems to work fine on `v0.5.2.post0`

**Comment by samuelsadok** [comment]
There was a bug where it says `min` instead of `max` (you can see it in the back trace), maybe that's what you're seeing.

Can you check if the issue still persists in the developer preview (`python3 -m pip install odrive --upgrade --pre`)?

**Comment by igor-markovic** [comment]
Thanks, I can confirm the issue is fixed in `v0.6.0.dev27`.
